app/src/sharedTest/java/org/oppia/app/topic/practice/TopicPracticeFragmentTest.kt

```diff
@@ -194,6 +194,21 @@ class TopicPracticeFragmentTest {
     onView(withId(R.id.topic_practice_start_button)).check(matches(isClickable()))
   }
 
+  @Test
+  fun testTopicPracticeFragment_loadFragment_configurationChange_TitleisCorrect() {
+    launchTopicActivityIntent(FRACTIONS_TOPIC_ID).use {
+      onView(
+        allOf(
+          withText(TopicTab.getTabForPosition(2).name),
+          isDescendantOfA(withId(R.id.topic_tabs_container))
+        )
+      ).perform(click())
+      onView(isRoot()).perform(orientationLandscape())
```

Comment:
Use this same code in this entire file for all other test cases too, meaning:

Replace this
```
activityScenario.onActivity { activity ->
      activity.requestedOrientation = Configuration.ORIENTATION_LANDSCAPE
}
```


with

```
onView(isRoot()).perform(orientationLandscape())
```

Reply:
Done :)